src/Tools/NUnitUtility/NUnitUtility.csproj

```diff
@@ -10,7 +10,7 @@
     <AssemblyName>NUnitUtility</AssemblyName>
 	<OutputPath>bin\$(Configuration)\</OutputPath>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
```

Comment:
@aparajit-pratap @QilongTang @pinzart90 does anyone know what the heck this thing is? Can we get rid of it?

Reply:
When this condition is not set, and we build for net6.0 we get warning about compiler not being able to resolve a conflict

![image](https://github.com/DynamoDS/Dynamo/assets/132492346/d548c527-293e-4087-96b8-1cb61d3d394a)
![image](https://github.com/DynamoDS/Dynamo/assets/132492346/35acc6d3-052b-4898-8a31-16c0a67b43f9)

Reply:
sorry @Cristhian-Marin-asdk I meant the nunitUtiity project and just dropping it, not the condition.

Reply:
Agree with Mike, let's remove this project.